js/web/lib/onnxjs/backends/webgl/ops/reshape-packed.ts

```diff
@@ -117,7 +115,7 @@ export class WebGLReshapePacked extends Reshape implements WebGLOperator {
   createRunData(handler: WebGLInferenceHandler, programInfo: ProgramInfo, inputs: Tensor[]): RunData {
     const inputTDs =
         [handler.getOrCreateTextureData(inputs[0], handler.getOrCreateTextureLayout(inputs[0], 1, false, [], false))];
-    let outputLayout = this.originalOutputLayout;
+    let outputLayout = this.outputLayout;
     if (outputLayout === undefined) {
       const originInputShape = inputs[0].dims;
       const outputShape = ShapeUtil.calculateReshapedDims(originInputShape, inputs[1].integerData);
```

Comment:
The *outputShape* has been calculated in createProgramInfo(). Any particular reason to do it again here?

Reply:
This is in case the op is not cached for the sequential runs, as we discussed offline for the case of conv-pack. 

I think with your PR's change, we should be able to get rid of it. But I would like to test it with a model first before removing it just to be safe. As it doesn't hurt to check the value here, I'll keep it as it is, wait till your PR's merged, test it out locally to ensure it works and then remove them.